datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py

```diff
@@ -172,6 +172,10 @@ def create_scraper_configuration(self, instance=None):
             instance.get('send_monotonic_counter', default_instance.get('send_monotonic_counter', True))
         )
 
+        config['send_counts_as_monotonic'] = is_affirmative(
```

Comment:
Since that's quite specific to distribution metrics. Should the name be a bit more precise like: `send_distribution_counts_as_monotonic` ?

Otherwise, this might be confused with sending all counts as monotonic.